bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -318,7 +321,13 @@ const isMeetingLocked = (id) => {
   return isLocked;
 };
 
-const setEmojiStatus = (userId) => { makeCall('setEmojiStatus', userId, 'none'); };
+const setEmojiStatus = (s) => {
```

Comment:
This parameter name seems dubious, could you change to something more meaningful?